### PR TITLE
Eviction settings cause false positive diskpressure and evict the pods

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1623329

### CODE REVIEW

**Comment by mburke5678** [comment]
@sjenning 
I updated the hard-eviction thresholds to the defaults in the Kubernetes doc. However, I didn't notice an default for `imagefs.inodesFree`. 
Do these changes apply for soft-eviction thresholds also?

**Comment by mburke5678** [comment]
@mdshuai PTAL

**Comment by mburke5678** [comment]
@sjenning PTAL

**Comment by sjenning** [comment]
LGTM

**Comment by mdshuai** [comment]
LGTM，thanks

**Comment by mburke5678** [comment]
@openshift/team-documentation Small change. PTAL

File: admin_guide/out_of_resource_handling.adoc [hunk 1]
```diff
@@ -101,27 +101,32 @@ For example:
 ----
 kubeletArguments:
   eviction-hard: <1>
-  - memory.available<500Mi<2>
-  - nodefs.available<500Mi
-  - nodefs.inodesFree<100Mi
-  - imagefs.available<100Mi
-  - imagefs.inodesFree<100Mi
+  - memory.available<100Mi<2>
```
Comment (kalexand-rh) [inline admin_guide/out_of_resource_handling.adoc]: I think you need a space before \<2\>.

File: admin_guide/out_of_resource_handling.adoc [hunk 2]
```diff
@@ -101,27 +101,32 @@ For example:
...
+  - imagefs.inodesFree<10% 
 ----
 
 <1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[hard eviction].
 <2> Eviction thresholds based on a specific eviction trigger signal.
 ====
 
+[NOTE]
+====
+The `inodesFree` values should always be a percentage. The other parameters can be percentage or a hard value.
```
Comment (kalexand-rh) [inline admin_guide/out_of_resource_handling.adoc]: s/should/must
s/hard/numerical ?

Maybe, "You must provide percentage values for `inodesFree` parameters but can provide percentage or numerical values for the other parameters."

Would this note make more sense before the examples?

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks, otherwise LGTM

**Pull request merged**